lib/pbench/test/unit/server/test_datasets_compare.py

```diff
@@ -77,7 +75,7 @@ def test_dataset_not_present(self, query_get_as, monkeypatch):
 
     def test_unsuccessful_get_with_incorrect_data(self, query_get_as, monkeypatch):
         def mock_extract(_tarball_path: Path, _path: str) -> str:
-            raise "IncorrectData"
+            return "IncorrectData"
```

Comment:
I'm pretty sure that all we need this mock to do is to keep the test from failing (e.g., to prevent it from actually looking for a real path within a real tarball); the behavior that this test is trying to drive comes from the `compare_csv_to_json()` mock.  So, as long as this mock does something innocuous, we're good.